rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -462,10 +473,24 @@ void gs_frame::HandleCursor(QWindow::Visibility visibility)
 	if (visibility == QWindow::Visibility::FullScreen && !m_show_mouse_in_fullscreen)
 	{
 		setCursor(Qt::BlankCursor);
+		m_mousehide_timer.stop();
 	}
 	else
 	{
 		setCursor(Qt::ArrowCursor);
+		if (m_hide_mouse_after_idletime)
```

Comment:
vertical spacing

Reply:
You want an empty line between the setCursor and the start of the if(..)?
Should the same empty line also go in the upper if between the setCursor and the timer stop?

Reply:
No. Imo scopes should be visually seperated from other stuff.
It's just for readability. You don't need to do it if you don't want to